src/Microsoft.ML.DataView/DataViewRowCursor.md

```diff
@@ -33,26 +33,27 @@ cursor until the `Batch` ID changes. Whereupon, the consolidator will find the
 next cursor with the next lowest batch ID (which should be greater, of course,
 than the `Batch` value we were just iterating on).
 
-Put another way: if we called `GetRowCursor` (possibly with an `IRandom`
+Put another way: suppose we called `GetRowCursor` (possibly with a `Random`
 instance), and we store all the values from the rows from that cursoring in
```

Comment:
>instance [](start = 0, length = 8)

batch

Reply:
This was put on `instance`, and that word is intentional, since this is a `Random` instance, that is, an instantiation of the `System.Random` class. However, I am going to try to rephrase, since there is clearly something confusing about that word in context.

Reply:
I've decided to go with "with an optional `Random` parameter" to indicate what I'm talking about, I think that helps?